install/install.sh

```diff
@@ -102,152 +78,19 @@ elevate_priv() {
   fi
 }
 
-install_dependencies() {
-  local sudo="$1"
-  confirm "Install dependencies on system?"
-  info "Install dependencies..."
-  printf '\n'
-  $sudo apt-get update
-  $sudo apt-get install --yes python3 
-  $sudo apt-get install --yes python3-pip 
-  printf '\n'
-  completed "Dependencies successful installed"
-}
-
-check_configuration_dir() {
-  local configuration_dir="$1"
-  local sudo="$2"
-
-  if [ ! -d "$CONFIGURATION_DIR" ]; then
-    warn "Configuration location $CONFIGURATION_DIR does not appear to be a directory"
-    confirm "Do you want to create the directory"
-    $sudo bash -c "mkdir "${CONFIGURATION_DIR}""
-    completed "Directory $CONFIGURATION_DIR created"
-  fi
-}
-
-create_default_configuration() {
-  local sudo="$1"
-  local path="$CONFIGURATION_DIR/wemulate.yml"
-  $sudo bash -c "cat > "${path}"" << EOF
----
-wemulate:
-  db_location: /etc/wemulate/wemulate.db
-EOF
-  completed "Default configuration $path is generated"
-}
-
-configure_mgmt_interface() {
-  local sudo="$1"
-  $sudo wemulate config set -m $INTERFACE --force 
-  completed "Management interface $INTERFACE is configured"
-}
-
 create_startup_configuration() {
-  local sudo="$1"
-  local path="$CONFIGURATION_DIR/startup.sh"
-  local conf_folder=$CONFIGURATION_DIR/config
+  info "Creating startup configuration"
   local cron_config_file=/etc/crontab
-  $sudo bash -c "cat > "${path}"" << EOF
-#!/bin/bash
-for directory in $conf_folder/*; do
-    bash \$directory/bridge.conf
-    bash \$directory/tc.conf
-done
-EOF
-  completed "Startup configuration $path is generated"
-  $sudo bash -c "cat >> "${cron_config_file}"" << EOF
-@reboot root    bash $path >> $cron_config_file
+  sudo bash -c "cat >> "${cron_config_file}"" << EOF
+@reboot root    wemulate restore device >> $cron_config_file
```

Comment:
The wemulate restore command is not implemented yet, so we have to stick with the current solution. Can you change that?

Reply:
Done